flickit-assessment-core/src/main/java/org/flickit/assessment/core/adapter/in/rest/evidence/GetEvidenceTypesResponseDto.java

```diff
@@ -0,0 +1,7 @@
+package org.flickit.assessment.core.adapter.in.rest.evidence;
+
+import org.flickit.assessment.core.application.port.in.evidence.GetEvidenceTypesUseCase;
```

Comment:
you can import `EvidenceTypeItem` to have shorter parameter list of record.

`import org.flickit.assessment.core.application.port.in.evidence.GetEvidenceTypesUseCase.EvidenceTypeItem;`

`public record GetEvidenceTypesResponseDto(List<EvidenceTypeItem> evidenceTypeItems) {}`